src/main/java/brave/opentracing/BraveSpan.java

```diff
@@ -44,7 +44,7 @@ static BraveSpan wrap(brave.Span span) {
     }
 
     private final brave.Span delegate;
-    private final SpanContext context;
+    private BraveSpanContext context;
```

Comment:
an aside, as this part of the change won't work anyway.. 

if we did do something like this, we'd end up with a new synchronization/thread visibility concern.